spec/support/search_helper.rb

```diff
@@ -1,9 +1,11 @@
 module SearchHelper
+  DEFAULT_INDEX = "Vacancy_test#{ENV.fetch('GITHUB_RUN_ID', '')}_publish_on_desc"
```

Comment:
This line - and indeed this file - is obsolete as of https://github.com/DFE-Digital/teacher-vacancy-service/commit/923c78946291cb99899d78ed09ff5f2d97987749. But we expect to reinstate it in the future, and when we do, the changes to this file will save someone some confusion.